### ISSUE TITLE
Update existing KB article: How to Audit the PDF Media Library

### ISSUE THREAD

**Issue by MDomngz** [department-of-veterans-affairs/va.gov-cms#18704]
## Background context
**What is the request for?**
- [ ] Create new article
  - [ ] I have reviewed the Knowledge Base to confirm that an article for this topic does not already exist
- [x] Update existing article: (https://prod.cms.va.gov/help/how-to-audit-the-pdf-media-library)

**Audience**
Who is this intended for? Who would need this information? (i.e. All editors, only editors for a particular product, Internal use only, etc.)
- PDF Audit workflow
- VAMC editors that have uploaded a PDF for story use

**Other related KB articles, if applicable:**
- [If you know that related KB articles already exist, link them here]

## Is the article associated with an ongoing product or feature update?
- [x] Yes [PDF Audit view\](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17718)
- [ ] No

  
## Priority or deadline for request
Choose either Deadline or a priority:

- [ ] Deadline: [name the deadline here]
- [x] Priority:
  - [ ] Critical - Without this, there is a large negative impact on the majority of editors causing negative consequences within the CMS; Should be completed as soon as possible
  - [ ] High - Without this, there is negative impact on some editors causing negative consequences within the CMS; should be completed within the current sprint
  - [x] Medium - Without this, there is confusion for editors but not causing negative consequences within the CMS; Should be completed within the next 1-2 sprints
  - [ ] Low - This is not currently causing confusion for many editors but would be a nice to have or is something that may be more impactful in the future; Should be completed within the next 3-4 sprints

## Acceptance Criteria
- [ ] Article is created/updated using [KB content style guidelines](https://va-gov.atlassian.net/wiki/spaces/VAGOV/pages/1771831299/Knowledge+Base+content+style) and saved as a draft
- [ ] Impacts on other KB articles have been considered and any necessary changes have been made and saved as draft
- [ ] If necessary, add link to the new KB article in the [Content model document](https://prod.cms.va.gov/admin/structure/cm_document) for that content type 
- [ ] [Ticket for article review](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&labels=Knowledge+Base%2C+Needs+refining&template=knowledge-base-article-review.md&title=Review+of+new+or+updated+KB+article+%3A+%3Cinsert+topic+here%3E) has been created

**Comment by MDomngz** [comment]
@gracekretschmer-metrostar 
A new ticket - while you were out we determined that the KB article around the PDF audit feature requires refinement. Specifically, we are focusing on the area where the audit UI references (Previous Version) in terms of being activated/published. 

Scenario: 
When an editor has detached their PDF from the content, if the PDF is part of the backup, it cannot be deleted. Instead, the UI will display "(Previous Version)," indicating that the backup for Drupal requires a copy of the PDF file.

I met with Edmund to review this functionality and it is working as expected. However, this may put a slight damper on our product outcome goals for the script work focusing on pushing out unattached PDFs. 

Additionally, I reviewed the findings with Erika to keep her updated on any adjustments required.
